package.json

```diff
@@ -50,8 +54,10 @@
     "envify": "^3.4.0",
     "eslint": "^3.3.1",
     "eslint-config-mourner": "^2.0.0",
+    "eslint-plugin-flowtype": "^2.20.0",
     "eslint-plugin-html": "^1.5.1",
     "github-slugger": "^1.1.1",
+    "flow-bin": "^0.2.3",
```

Comment:
Ah! Thanks @davidtheclark. I stupidly changed the package name without updating the version.